styles.css

```diff
@@ -4,7 +4,7 @@
   .body {
     position: relative;
     width: 375px;
-    height: 5410px;
+    height: 5610px;
```

Comment:
:red_circle: I suggest removing the height property from the body element since it will adjust automatically to the size of its contents. This will ensure the design matches the Figma design.
|Your Design|Figma Design|
|:---------------:|:----------------:|
|![Screenshot from 2023-01-24 23-03-47](https://user-images.githubusercontent.com/26736582/214417527-09259f22-b6c7-4ad0-a938-387f1d4c504e.png)|![Screenshot from 2023-01-24 23-14-16](https://user-images.githubusercontent.com/26736582/214417901-61f6b996-0324-445d-9069-76ee723494a4.png)|

Reply:
> Observation

I found the observation curious and I would like to make the observation that template 1 has been updated in figma and not in the github repository that contains the instructions.

Consequently I will fix the `<p>` tag in the button but I will keep the updated figma design (button on the left border).

|Form|Answers|
|:---------------:|:----------------:|
|![imagen](https://user-images.githubusercontent.com/90431831/214446611-bd4ab379-ec8d-476e-92de-8f500e621aad.png)|![imagen](https://user-images.githubusercontent.com/90431831/214446923-1c5e039a-fe75-4d64-a0e3-bedaab00643a.png)|